TRADEMARKS.md

```diff
@@ -0,0 +1,231 @@
+- You must contact us and obtain prior written approval. See section "Where to Get Further Information" for our contact information. This review process helps ensure that hardware products associated with AmpliPi adhere to open source principles, benefit the AmpliPi community and our contributors, and are compliant with the AmpliPi trademark policies, software license, and applicable laws.
+
+## Uses we consider infringing without seeking further permission from us
+
+We will likely consider using the Marks in a software distribution that combines our software with any other software program an infringement of our Marks. We would consider your software "combined" with ours if you create a single binary for both software programs, as well as if by installing our software it automatically installs yours. We will likely not consider your software "combined" with ours if it is on the same media but requires separate, independent action to install it. 
+
+Having said that, if you have new ideas for integrating AmpliPi with other software, please reach out to us for further discussion. We are open to new ideas that are of benefit to the AmpliPi project, our contributors, and to the open source community as a whole. Our contact information can be found in the section "Where to Get Further Information".
+
+# Use for Non-Software Goods and Services
+
+See "Universal Considerations for All Uses", above, which also apply. 
+
+## Uses we consider non-infringing
+
+### Websites
+
+You may use the Word Marks and Logos, but not the Trade Dress, on your webpage to show your support for the Project as long as:
+
+- The website has branding that is easily distinguished from the Project Trade Dress;
+- You own branding or naming is more prominent than any Project Marks;
+- The Logos hyperlink to the Project website;
+- The site does not mislead customers into thinking that either your website, service, or product is our website, service, or product; and
+- The site clearly states that you are not affiliated with or endorsed by the Project.
+
+### Publications and presentations
+
+You can use the Word Marks in book and article titles, and the Logo in illustrations within the document, as long as the use does not suggest that we have published, endorse, or agree with your work. 
+
+## Uses for which we are granting a license
+
+### User groups
+
+You can use the Word Marks as part of your user group name provided that: 
+
+- The main focus of the group is the software;
+- Any software or services the group provides are without cost;
+- The group does not make a profit;
+- Any charge to attend meetings are to cover the cost of the venue, food and drink only.
+
+Note that the Universal considerations for all uses, above, still apply, specifically, that you may not use or register the Marks as part of your own trademark, service mark, domain name, company name, trade name, product name or service name. 
```

Comment:
"Universal considerations for all uses, above, still apply, specifically," do we need this many commas? Also, the Universal Considerations for All Uses should have its capitalization

Reply:
Those commas were part of the original wording, but I do not believe we need so many. How about this:

_"Note that the Universal considerations for all uses above still apply, specifically that you may not use or register the Marks as part of your own trademark, service mark, domain name, company name, trade name, product name or service name."_